glean-core/ffi/examples/glean_app.c

```diff
@@ -11,6 +11,7 @@ int main(void)
   FfiConfiguration cfg = {
     "/tmp/glean_data",
     "c-app",
+    "C",
```

Comment:
```suggestion
    "C Sample",
```

Reply:
Ah, I thought this wouldn't apply after the changes. This raises the question if we should be more specific for the preview, benchmark, example. They are all "Rust". Should each be "Rust Preview", "Rust Benchmark" etc. 

Personally I think this adds subjectivity and goes against what we expect this "language_binding_name" to be. On the other hand could be useful. I vote for not doing it, what do you think? That would also mean this wouldn't be "C Sample", but "C".

Reply:
Mh, you're correct. Let's go with `C`.